backend/api/auth/src/main/java/co/airy/core/api/auth/controllers/UsersController.java

```diff
@@ -161,50 +156,23 @@ private String getResetToken(String userId) {
         return jwt.tokenFor(userId, refreshClaim);
     }
 
-    //TODO: Write a custom ExceptionHandler for JDBI
-    @PostMapping("/users.invite")
-    ResponseEntity<InviteUserResponsePayload> inviteUser(@RequestBody @Valid InviteUserRequestPayload inviteUserRequestPayload) {
-        final UUID id = UUID.randomUUID();
-        final Instant now = Instant.now();
-
-        invitationDAO.insert(Invitation.builder()
-                .id(id)
-                .acceptedAt(null)
-                .createdAt(now)
-                .email(inviteUserRequestPayload.getEmail())
-                .sentAt(null)
-                .updatedAt(now)
-                .createdBy(null)
-                .build());
-        return ResponseEntity.status(HttpStatus.CREATED).body(InviteUserResponsePayload.builder()
-                .id(id)
-                .build());
-    }
-
-    @PostMapping("/users.accept-invitation")
-    ResponseEntity<?> acceptInvitation(@RequestBody @Valid AcceptInvitationRequestPayload payload) {
-        final Invitation invitation = invitationDAO.findById(payload.getId());
+    @PostMapping("/users.list")
+    ResponseEntity<ListResponsePayload> listUsers() {
+        final List<User> users = userDAO.list();
 
-        if (!invitationDAO.accept(invitation.getId(), Instant.now())) {
-            return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(new EmptyResponsePayload());
-        }
-
-        final User user = User.builder()
-                .id(UUID.randomUUID())
-                .email(invitation.getEmail())
-                .firstName(payload.getFirstName())
-                .lastName(payload.getLastName())
-                .passwordHash(passwordService.hashPassword(payload.getPassword()))
-                .build();
+        ListResponsePayload listResponsePayload = new ListResponsePayload();
 
-        userDAO.insert(user);
+        listResponsePayload.setData(users
+                .stream()
+                .map(u -> UserPayload.builder()
+                        .firstName(u.getFirstName())
+                        .lastName(u.getLastName())
+                        .token(null)
```

Comment:
Just for my understanding: Do we have to set this to null explicitly?

Reply:
not that I know, I thought the clarity wouldn't hurt though :)

If this grows again, it's probably not good to use this `UserPayload` for all the responses... I thought it was a little overkill right now and that's my way of "documenting" it. (I can/should leave a comment maybe?)